packages/salesforcedx-vscode-apex/src/embeddedSoql/soqlCompletion.ts

```diff
@@ -93,7 +94,7 @@ export const soqlMiddleware: Middleware = {
     const items: ProtocolCompletionItem[] = Array.isArray(apexCompletionItems)
       ? (apexCompletionItems as ProtocolCompletionItem[])
       : ((apexCompletionItems as CompletionList)
-          .items as ProtocolCompletionItem[]);
+        .items as ProtocolCompletionItem[]);
```

Comment:
oh boy.  as casting.  We might just need a WI to address not doing this here.  Seems dangerous to convert from the type VSCode is telling us to the type inside the LSP